### DISCUSSION TITLE
Funktionsweise der Kosmodrome, Bedarf unklar

### DISCUSSION THREAD

**Discussion by FrlPetronella** [Taludas/Anno2205-NewFrontiers#21] [Q&A]
Mir ist die Funktionsweise der Kosmodrome in 1.0.6 unklar. Das Kosmodrom zeigt 100% Auslastung, "produziert" aber keine Astronauten. Als Bedarf wird angegeben "lokale Ressource, unbegrenzt verfügbar". Was ist damit gemeint? Was benötigt das Kosmodrom tatsächlich? Bestimmte Bevölkerungsstufen? Müssen diese voll versorgt sein? Oder teilweise? Oder mit bestimmten Gütern?

**Comment by Taludas** [comment]
Die Astronautenlager funktionieren immer noch genau so wie im vanilla Spiel. Nur das UI würde ausgetauscht, um die Zusatzwarenproduktion der Orbitallagertechnologie abzubilden. 
Die Funktionsweise ist immer noch wie vorher, sprich die Menge an Bevölkerung im Radius beeinflusst die Produktivität des Gebäudes. Die Menge an Astronautenoutput ist leider nicht zu ändern. Die korrekte Menge an produzierten Astronauten sieht man oben in der Leiste, wo die Zeit herunterläuft bis zum nächsten Versorgungsflug. Wenn man dort drüber hovert, sieht man die aktuelle Produktion und den Bedarf der Station.

**Comment by FrlPetronella** [comment]
Ich habe inzwischen begriffen, dass die Ausbildungszentren nur dann Astronauten hervorbringen, wenn die Einwohner im Umkreis voll versorgt sind. Dabei scheint auch die Bevölkerungsstufe eine Rolle zu spielen. Arbeiter z. B. "produzieren" keine Astronauten, auch wenn die voll versorgt sind.

Insgesamt ist es für mich aber immer noch schleierhaft, wie man die Ausbeute gezielt beeinflussen kann. Zur Zeit habe ich bei Biologen eine Sektorbilanz von 41. Vorher war es 23. Die Erhöhung kam zustande, als ich EW von Managern zu Investoren upgegradet habe. Wird die Zahl der Astronauten von der nackten EW-Zahl oder auch von deren Bevölkerungsstufe beeinflusst?

Die "Sektorbilanz" 41 wird mit dabei in jedem gemäßigten Sektor angezeigt, egal wie viele EW und wie viele Kosmodrome es gibt. Es ist also eigentlich eine "Weltbilanz".

Im Originalspiel benötigt man einfach 500 EW im Umkreis des Kosmodroms.

![screenshot_2024-06-16-22-54-25](https://github.com/Taludas/Anno2205-NewFrontiers/assets/166033711/5b818888-99b4-4a6f-a33c-0fc90988232e)
![screenshot_2024-06-23-21-12-27](https://github.com/Taludas/Anno2205-NewFrontiers/assets/166033711/60075663-653a-43eb-b250-5d1c52b16cae)

Ich bin mir auch nicht sicher, ob der Bedarf an Astronauten insgesamt wirklich ausgewogen ist. Da jetzt auch die Module der Orbitalstation Astronauten benötigen, ist der Bedarf deutlich höher. Meine Orbitalstation ist nicht voll ausgebaut. Aber ich brauche so viele Astronauten, dass ich die Stadt um die Kosmodrome plane wie in Anno 1800 um Rathäuser oder Handelskammern. Kann sein, dass sich das mit mehr Bevölkerung gibt, wenn man den Nexus für sich austariert hat und keine weiteren Astronauten benötigt

**Reply by Taludas** [reply]
Die Anzahl der ausgebildeten Astronauten hängt ganz simpel von der absoluten Anzahl von aktuellen Einwohnern im angezeigten Radius des Gebäudes ab. Daher auch deine Beobachtungen, dass sowohl Upgraden als auch Versorgen zu mehr Astronauten führte, denn dadurch ist die Anzahl der Bewohner im Umkreis gestiegen. Funktioniert im Endeffekt genau so wie die Postämter/Briefkästen in Anno 1800.
Für 100% Produktivität sind nötig:
- auf der Erde 10000 Einwohner
- in der Arktis: 2500 Einwohner
- in der Tundra/auf dem Mond: 1000 Einwohner.

Die Anzahl an benötigten Ausbildungszentren ist deutlich angestiegen. Im vanilla Spiel brauchtest du genau 2 voll ausgebaute Raumzentren, um alle fünf Workshops eines Fachgebiets zu versorgen. Ernsthaft? 
Besonders die Astronauten aus den weniger dicht besiedelten Regionen sind jetzt deutlich mehr gefragt, sodass durchaus die Relevanz der Zentren angestiegen ist. Das wird auch durch die Nexustechnologie Stufe 5 Schwerindustrie unterstrichen, mit denen man aus diesen Gebäuden zusätzliche Endprodukte generieren kann - ohne Eingangswaren! Es könnte sich also definitiv lohnen, die Gebäude verstärkt in die Stadtplanung mit einzubeziehen.
